apps/emqx_message_transformation/src/emqx_message_transformation.app.src

```diff
@@ -0,0 +1,15 @@
+{application, emqx_message_transformation, [
+    {description, "EMQX Message Transformation"},
+    {vsn, "0.1.0"},
+    {registered, [emqx_message_transformation_sup, emqx_message_transformation_registry]},
+    {mod, {emqx_message_transformation_app, []}},
+    {applications, [
+        kernel,
```

Comment:
depend on `emqx_schema_registry` ?

Reply:
D'oh, forgot to add it here, too.  I'll add it in the follow up where I add the test endpoint.  Thanks!  :see_no_evil: